.github/workflows/update-sync-param-files.yaml

```
@@ -0,0 +1,35 @@
+name: update-sync-param-files
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  sync-param-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - uses: actions/checkout@v1
+
+      - name: Generate sync-param-files.yaml
+        run: |
+          git clone https://github.com/autowarefoundation/autoware.universe.git -b main /tmp/autoware.universe
```

If you suppose the default branch, removing the branch name can reduce maintenance costs.

```suggestion
          git clone https://github.com/autowarefoundation/autoware.universe.git /tmp/autoware.universe
```

---

Please check this commit
https://github.com/autowarefoundation/autoware_launch/pull/75/commits/bd49be9fdc1248a9bd4c775e5e8769e9af44ea1c